<launch>
  <arg name="odom_topic" default="odometry_merged"/>
  <arg name="child_frame_id" default="base_footprint"/>

  <node name="odom2tf" pkg="leo_navigation" type="odom2tf">
    <param name="odom_topic" value="$(arg odom_topic)"/>
    <param name="child_frame_id" value="$(arg child_frame_id)"/>
  </node>

</launch>